test/e2e/handler/nns_update_timestamp_test.go

```diff
@@ -28,22 +26,28 @@ var _ = Describe("[nns] NNS LastSuccessfulUpdateTime", func() {
 			originalNNSs[node] = nodeNetworkState(key)
 		}
 	})
-	Context("when network configuration hasn't change", func() {
-		It("should not be updated", func() {
+	Context("when network configuration is not changed by a NNCP", func() {
+		FIt("should be updated only if configuration changed", func() {
+			timeout := 2 * nmstatenode.NetworkStateRefresh
+			By(fmt.Sprintf("Sleeping %s to wait for nns refresh", timeout))
```

Comment:
Can't you use Consistently instead? So we have an early exit in case the functionality does not work as expected.

Reply:
Good idea, done.